apps/www/_blog/2023-12-01-automatic-cli-login.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Automatic CLI login'
+description: 'Explore the technical implementation and security measures behind CLI's new automatic login feature.'
```

Comment:
The sneaky problem here is that the `'` in CLI's conflicts with the opening and closing `'`s.
The solution is to open and close with `"`. @RamiroND